File: web/score/src/org/me/models/Product.java
package org.me.models;

import java.math.BigDecimal;

public class Product {

	public String id;
	public String username;
	public String title;
	public String imagePath;
	public String content;
	public BigDecimal cash;
	public String qrCode;

	public static Product generateProduct(String a) {
		if (a == null)
			a = "a";
		Product po = new Product();
		po.title = "product " + a;
		po.imagePath = "image " + a + " path";
		po.content = "content " + a;
		po.cash = new BigDecimal(100);
		po.qrCode = "qrCode " + a;
		return po;
	}

}

File: client/score.client/score.client/Models/Product.cs
using System;
using System.Net;
using System.Windows;
using System.Windows.Controls;
using System.Windows.Documents;
using System.Windows.Ink;
using System.Windows.Input;
using System.Windows.Media;
using System.Windows.Media.Animation;
using System.Windows.Shapes;

namespace score.client.Models
{
    public class Product
    {
        public String id { get; set; }
        public String title { get; set; }
        public String imagePath { get; set; }
        public String content { get; set; }
        public Decimal cash { get; set; }
        public String qrCode { get; set; }

    }
}
